src/utils/NXTHelpers.cpp

```diff
@@ -162,4 +163,21 @@ namespace utils {
         return device.CreatePipelineLayout(&descriptor);
     }
 
+    nxt::BindGroupLayout MakeBindGroupLayout(
+        const nxt::Device& device,
+        std::initializer_list<nxt::BindGroupBinding> bindingsInitializer) {
+        std::vector<nxt::BindGroupBinding> bindings;
+        nxt::ShaderStageBit kNoStages{};
+        for (const nxt::BindGroupBinding& binding : bindingsInitializer) {
+            if (binding.visibility != kNoStages) {
```

Comment:
`MakeBindGroupLayout` now uses a `visibility` of `0` to indicate bindings that should be skipped.